app/models/microservices_engine/connection.rb

```diff
@@ -31,7 +31,7 @@ def get(path, params = {})
 
     def self.get(resource, path, params = {})
       conn = Connection.where(object: resource.to_s) # Does :abc match "abc"?
-      conn ? conn.get(path, params) : {}
+      conn.get(path, params) if conn
```

Comment:
That's because of space characters holster a "value" so `' '` would evaluate `true`. The others still work equally. But yes true, that is a case where it is difference.